### ISSUE TITLE
Accessibility: using voiceover, cannot swipe down through headings in SingleChildScrollView

### ISSUE THREAD

**Issue by jolamar** [flutter/flutter#102653]
When the heading is off screen, using voice over to swipe down through the headings, it says "Heading not found".

**Comment by darshankawar** [comment]
@jolamar 
Can you provide `flutter doctor -v` and a [complete minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) ?
Thanks.
